src/app/components/elements/TagList.jsx

```diff
@@ -32,10 +32,15 @@ export default ({ post, horizontal, single }) => {
     // Category should always be first.
     tags.unshift(post.category);
 
-    // Uniqueness filter.
-    tags = tags.filter(
-        (value, index, self) => value && self.indexOf(value) === index
-    );
+    tags = tags
+        .filter(
+            // Uniqueness filter.
+            (value, index, self) => value && self.indexOf(value) === index
+        )
```

Comment:
I guess I should put the uniqueness filter into the named function and use that here.